src/components/Choropleth.vue

```diff
@@ -41,13 +42,14 @@ const backgroundLayer = {
     minZoom: 3
 };
 
-const map = ref<typeof LMap | null>(null);
+const map = shallowRef<typeof LMap | null>(null);
 const bounds = ref<Bounds | null>(null);
-const geoJsonLayer = ref<GeoJSON<FeatureProperties, Geometry>>(geoJSON(undefined));
+const geoJsonLayer = shallowRef<GeoJSON<FeatureProperties, Geometry>>(geoJSON(undefined));
+const countryOutlineLayer = shallowRef<Polyline | null>(null);
```

Comment:
same with these layers, only need shallow refs because we just destroy them and replace them every update